### ISSUE TITLE
MD037 not respected in table

### ISSUE THREAD

**Issue by Spixmaster** [rvben/rumdl#759]
### rumdl version

0.2.43

### Installation method

Other

### Bug description

```md
# Konten

| Webseite    | E-Mail-Adresse | Notiz | Konfiguration aktuell |
| :---------- | :------------- | :---- | :-------------------- |
| **Archiv ** |                |       |                       |

**h **
```

Issues:

- `MD037: Spaces inside emphasis markers: "**h **"`

The issue for `**Archiv **` is not detected.

### Steps to reproduce

See the description.

### Expected behavior

`**Archiv **` is an an error due to the space.

### Actual behavior

No error.

### Configuration

```toml

```

### Operating system

Linux

**Comment by rvben** [comment]
Fixed in v0.2.44.

MD037 and MD052 did not look inside table cells at all, so `**Archiv **` in a cell went unchecked while the identical `**h **` in a paragraph was reported. Both are flagged now, and both are fixable. Your exact input on 0.2.46:

```console
$ rumdl check t.md
t.md:5:3: [MD037] Spaces inside emphasis markers: "**Archiv **" [*]
t.md:7:1: [MD036] Emphasis used instead of a heading: 'h '
t.md:7:1: [MD037] Spaces inside emphasis markers: "**h **" [*]

Issues: Found 3 issues in 1 file (7ms)
Run `rumdl fmt` to automatically fix 2 of the 3 issues
```

Thanks for the report.
